diff --git a/node_modules/next/dist/esm/server/lib/incremental-cache/index.js b/node_modules/next/dist/esm/server/lib/incremental-cache/index.js
index cc7d3fb..09bf337 100644
--- a/node_modules/next/dist/esm/server/lib/incremental-cache/index.js
+++ b/node_modules/next/dist/esm/server/lib/incremental-cache/index.js
@@ -244,9 +244,9 @@ export class IncrementalCache {
     async set(pathname, data, revalidateSeconds, fetchCache, fetchUrl, fetchIdx) {
         if (this.dev && !fetchCache) return;
         // fetchCache has upper limit of 2MB per-entry currently
-        if (fetchCache && JSON.stringify(data).length > 2 * 1024 * 1024) {
+        if (fetchCache && JSON.stringify(data).length > 100 * 1024 * 1024) {
             if (this.dev) {
-                throw new Error(`fetch for over 2MB of data can not be cached`);
+                throw new Error(`fetch for over 100MB of data can not be cached`);
             }
             return;
         }
diff --git a/node_modules/next/dist/server/lib/incremental-cache/index.js b/node_modules/next/dist/server/lib/incremental-cache/index.js
index bc11e84..042f55e 100644
--- a/node_modules/next/dist/server/lib/incremental-cache/index.js
+++ b/node_modules/next/dist/server/lib/incremental-cache/index.js
@@ -271,9 +271,9 @@ class IncrementalCache {
     async set(pathname, data, revalidateSeconds, fetchCache, fetchUrl, fetchIdx) {
         if (this.dev && !fetchCache) return;
         // fetchCache has upper limit of 2MB per-entry currently
-        if (fetchCache && JSON.stringify(data).length > 2 * 1024 * 1024) {
+        if (fetchCache && JSON.stringify(data).length > 100 * 1024 * 1024) {
             if (this.dev) {
-                throw new Error(`fetch for over 2MB of data can not be cached`);
+                throw new Error(`fetch for over 100MB of data can not be cached`);
             }
             return;
         }
diff --git a/node_modules/next/next-swc-fallback/@next/swc-darwin-arm64/README.md b/node_modules/next/next-swc-fallback/@next/swc-darwin-arm64/README.md
new file mode 100644
index 0000000..10a0ca9
--- /dev/null
+++ b/node_modules/next/next-swc-fallback/@next/swc-darwin-arm64/README.md
@@ -0,0 +1,3 @@
+# `@next/swc-darwin-arm64`
+
+This is the **aarch64-apple-darwin** binary for `@next/swc`
diff --git a/node_modules/next/next-swc-fallback/@next/swc-darwin-arm64/next-swc.darwin-arm64.node b/node_modules/next/next-swc-fallback/@next/swc-darwin-arm64/next-swc.darwin-arm64.node
new file mode 100644
index 0000000..db3cb97
Binary files /dev/null and b/node_modules/next/next-swc-fallback/@next/swc-darwin-arm64/next-swc.darwin-arm64.node differ
diff --git a/node_modules/next/next-swc-fallback/@next/swc-darwin-arm64/package.json b/node_modules/next/next-swc-fallback/@next/swc-darwin-arm64/package.json
new file mode 100644
index 0000000..511aca7
--- /dev/null
+++ b/node_modules/next/next-swc-fallback/@next/swc-darwin-arm64/package.json
@@ -0,0 +1,23 @@
+{
+  "name": "@next/swc-darwin-arm64",
+  "version": "13.4.12",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vercel/next.js",
+    "directory": "packages/next-swc/crates/napi/npm/darwin-arm64"
+  },
+  "os": [
+    "darwin"
+  ],
+  "cpu": [
+    "arm64"
+  ],
+  "main": "next-swc.darwin-arm64.node",
+  "files": [
+    "next-swc.darwin-arm64.node"
+  ],
+  "license": "MIT",
+  "engines": {
+    "node": ">= 10"
+  }
+}
\ No newline at end of file
